### DISCUSSION TITLE
Trust wallet not connecting

### DISCUSSION THREAD

**Discussion by rbyf24** [WalletConnect/walletconnect-monorepo#4562] [Developer Support]
Hello, not sure why but my trust wallet gets unknown error when connecting to a dexes through wallet connect. Has anyone had this problem? It happens when trying to connect via phone or laptop. I have cleared cache on all browsers, reinstalled trust wallet and made sure its the latest version. The wallet connects to dexes directly but anything through wallet connect just has the unknown error

**Comment by glitch-txs** [comment]
Hi! Thanks for reporting. Could you share the link to the websites where this error occurs?

Also, are you using Android or iOS? Thanks!

**Comment by rbyf24** [comment]
Thanks for your reply. I'm trying to connect it to https://jup.ag/swap. I am using Android

**Comment by rbyf24** [comment]
![Screenshot_20240525_161115_Chrome](https://github.com/WalletConnect/walletconnect-monorepo/assets/170814214/fd059f59-76b3-4e25-b808-4da89ec2d66e)
This is what I get for some context

**Comment by rbyf24** [comment]
UPDATE. Looks like it's a problem with my phone and not wallet connect as I tried on my older device and it works fine
